modular_skyrat/modules/cargo/code/items/improvedRCD.dm

```diff
@@ -0,0 +1,11 @@
+// Slightly improved version of the normal RCD, mostly as an engineering 'I got hella bread' purchase
+/obj/item/construction/rcd/combat/improved
```

Comment:
Why is it a subtype of the combat RCD, exactly?

Reply:
![image](https://user-images.githubusercontent.com/62520989/180626848-02cd64bb-d756-40a5-b15b-fc2d0c5e89ca.png)

real footage of my brain making this at 5am